modules/combined/tests/phase_field_fracture/crack2d.i

```diff
@@ -48,18 +48,20 @@
     G0_var = 'G0_pos'
     dG0_dstrain_var = 'dG0_pos_dstrain'
   [../]
+  [./DynamicTensorMechanics]
```

Comment:
Did you intend to use DynamicTensorMechanics here instead of TensorMechanics?  I suppose either one works.